core/templates/dev/head/components/state-editor/state-responses-editor/state-responses.directive.ts

```diff
@@ -197,6 +205,14 @@ oppia.directive('stateResponses', [
             }
           };
 
+          $scope.onChangeSolicitAnswerDetails = function() {
+            $scope.onSaveSolicitAnswerDetails(
+              $scope.stateSolicitAnswerDetailsService.displayed);
+            StateSolicitAnswerDetailsService.displayed = (
```

Comment:
is this required? If yes then, please move it before calling `onSaveSolicitAnswerDetails`.

btw, I don't think you'll have to change the displayed value as `$scope.stateSolicitAnswerDetailsService` is the `StateSolicitAnswerDetailsService`.

Reply:
Done